Lib/test/test_base64.py

```diff
@@ -545,6 +547,17 @@ def test_b85encode(self):
         self.check_other_types(base64.b85encode, b"www.python.org",
                                b'cXxL#aCvlSZ*DGca%T')
 
+    def test_b85encode_large_input(self):
+        # more than 512 bytes (+ padding)
+        large_input = string.ascii_letters.encode() * 12 + b"foo"
+        result = base64.b85encode(large_input)
+
+        # since the result is to large to fit inside a test,
+        # use a hash method to validate the test
+        self.assertEqual(len(result), 784)
+        self.assertEqual(hashlib.md5(result).hexdigest(),
```

Comment:
I'm not sure md5 is always available. WDYT @gpshead ?

Reply:
Good catch, forgot about this
According to https://docs.python.org/3/library/hashlib.html#hashlib.algorithms_guaranteed md5 may not be present, I'll switch to a sha1